src/PlayerContext.ts

```diff
@@ -0,0 +1,4 @@
+export default interface PlayerContext {
+    title: String;
```

Comment:
I think you'll want `string` (lowercase) instead of `String` as the former is the primitive.